bool manual_steer
bool right_lane
bool manual_v
bool stop
float64 command_v